Model/Config/Source/TokenizedPaymentMethods.php

```diff
@@ -33,7 +33,12 @@ public function toOptionArray()
             [
                 'value' => PaymentMethods\SepaPaymentMethod::TX_VARIANT,
                 'label' => PaymentMethods\SepaPaymentMethod::NAME
-            ],
+            ]
+            /** TODO: Enable recurring for Twint once the issue fixed on LPM side */
```

Comment:
I would suggest keeping this PR as a draft until the company wide issue is solved for Twint recurring.

Reply:
it is solved, vault_payment_token is getting populated and I can see the token created in Adyen CA